### PR TITLE
[author][new]Added jquery-jsonview@1.2.3

### PR DESCRIPTION
**Pull request by yesmeck**

### CODE REVIEW

File: ajax/libs/jquery-jsonview/package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        "url": "https://github.com/jquery/jquery-color/blob/2.1.2/MIT-LICENSE.txt"
+      }
+    ],
+    "docs": "https://github.com/yesmeck/jquery-jsonview#readme",
+    "demo": "http://blog.yesmeck.com/jquery-jsonview/",
+    "dependencies": {
+      "jquery": "*"
+    }
+  },
+  "bowerJSON": {
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-jsonview/package.json]: Please remove `"bowerJSON"` section.

File: ajax/libs/jquery-jsonview/package.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  "keywords": [
+    "json"
+  ],
+  "author": "Meck <yesmeck@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/yesmeck/jquery-jsonview/issues"
+  },
+  "homepage": "https://github.com/yesmeck/jquery-jsonview",
+  "jqueryJSON": {
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-jsonview/package.json]: Please remove whole the `"jqueryJSON"` section.

File: ajax/libs/jquery-jsonview/package.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  "npmName": "jquery-jsonview",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/index.html"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-jsonview/package.json]: `"filename"` should be `"jquery.jsonview.min.js"`

File: ajax/libs/jquery-jsonview/package.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      "node_modules",
+      "bower_components",
+      "test",
+      "tests"
+    ],
+    "dependencies": {
+      "jquery": "*"
+    }
+  },
+  "dependencies": {},
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-jsonview/package.json]: Please remove empty `dependencies`

**Comment by yesmeck** [comment]
@PeterDaveHello @maruilian11 Fixed and pushed.

File: ajax/libs/jquery-jsonview/package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-jsonview",
+  "version": "1.2.3",
+  "description": "jQuery plugin for display pretty JSON.",
+  "main": "dist/jquery.jsonview.js",
```
Comment (maruilian11) [inline ajax/libs/jquery-jsonview/package.json]: image
please remove `main`, thanks!

Comment (yesmeck) [inline ajax/libs/jquery-jsonview/package.json]: Removed. CI passed now.

**Comment by maruilian11** [comment]
@PeterDaveHello i think this PR is ok.

File: ajax/libs/jquery-jsonview/package.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  ],
+  "author": "Meck <yesmeck@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://github.com/yesmeck/jquery-jsonview",
+  "npmName": "jquery-jsonview",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-jsonview/package.json]: I think it should be `"**/*.+(css|js)"` or `"**/*.+(css|js|map)"`, there is a html file in `dist`

**Comment by PeterDaveHello** [comment]
Since the parent of the commit in this PR is a little bit old, please also do me a favor to rebase this branch with our latest master branch, thanks.

**Comment by yesmeck** [comment]
@PeterDaveHello fixed and rebased.

**Comment by PeterDaveHello** [comment]
hmmm ... seems not rebased with our latest master branch, anyway, I'll merge it this time, thanks!

**Comment by yesmeck** [comment]
:fireworks: Thanks.

**Comment by PeterDaveHello** [comment]
@yesmeck would you like to mention the CDN resource on the readme or website? Thanks.

**Comment by yesmeck** [comment]
I'll add CDN link on readme

**Comment by PeterDaveHello** [comment]
Great, it'll be available on the website and cdn server within 30 minutes.

**Pull request merged**